File: app/src/main/java/zjm/cst/dhu/readattitude/mvp/view/MainContract.java
package zjm.cst.dhu.readattitude.mvp.view;

import zjm.cst.dhu.library.mvp.presenter.BasePresenter;
import zjm.cst.dhu.library.mvp.view.base.BaseView;

/**
 * Created by zjm on 2017/5/9.
 */

public interface MainContract {
    interface View extends BaseView {

    }

    interface Presenter extends BasePresenter<View> {

    }
}

File: app/src/main/kotlin/zjm/cst/dhu/readattitude/mvp/view/MainContract.kt
package zjm.cst.dhu.readattitude.mvp.view

import zjm.cst.dhu.library.mvp.presenter.BasePresenter
import zjm.cst.dhu.library.mvp.view.base.BaseView

/**
 * Created by zjm on 2017/5/9.
 */

interface MainContract {
    interface View
        : BaseView

    interface Presenter
        : BasePresenter<View>
}
